ocaml/tests/test_repository_helpers.ml

```diff
@@ -1163,13 +1090,18 @@ module EvalGuidanceForOneUpdate = Generic.MakeStateless (struct
                       id= "UPDATE-0000"
                     ; summary= "summary"
                     ; description= "description"
-                    ; rec_guidance= Some Guidance.RebootHost
+                    ; rec_guidance= Some RebootHost
                     ; abs_guidance= None
+                    ; guidance=
+                        [
+                          (Recommended, [RebootHost])
```

Comment:
I think for a valid update, the `RebootHost` guidance here should be a `Mandatory` one?